protocol/features.json

```diff
@@ -679,7 +679,9 @@
       "0056-REWA-216",
       "0056-REWA-217",
       "0056-REWA-218",
-      "0056-REWA-219"
+      "0056-REWA-219", 
+      "0056-REWA-220", 
+      "0056-REWA-221"
```

Comment:
These need to go in a new feature section, otherwise they will come under reward scaling.